src/WebApiContrib.Core.Formatter.Protobuf/ProtobufFormatterOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using System.Collections.Generic;
+
+namespace WebApiContrib.Core.Formatter.Protobuf
+{
+    public class ProtobufFormatterOptions
+    {
+        //public IFormatterResolver FormatterResolver { get; set; } = ContractlessStandardResolver.Instance;
+
+        public HashSet<string> SupportedContentTypes { get; set; } = new HashSet<string> { "application/x-protobuf", "application/x-protobuf" };

```

Comment:
personally `application/octet-stream` is a bit invasive I think for a default option, but add it if it makes sense to you